src/main/java/uk/nhs/digital/nhsconnect/lab/results/model/edifact/segmentgroup/LabResult.java

```diff
@@ -0,0 +1,89 @@
+package uk.nhs.digital.nhsconnect.lab.results.model.edifact.segmentgroup;
+
+import static java.util.stream.Collectors.toList;
+
+import java.util.List;
+import java.util.Optional;
+
+import lombok.Getter;
+import lombok.NonNull;
+import uk.nhs.digital.nhsconnect.lab.results.model.edifact.DiagnosticReportCode;
+import uk.nhs.digital.nhsconnect.lab.results.model.edifact.LaboratoryInvestigation;
+import uk.nhs.digital.nhsconnect.lab.results.model.edifact.LaboratoryInvestigationResult;
+import uk.nhs.digital.nhsconnect.lab.results.model.edifact.MessageTrailer;
+import uk.nhs.digital.nhsconnect.lab.results.model.edifact.SequenceDetails;
+import uk.nhs.digital.nhsconnect.lab.results.model.edifact.SequenceReference;
+import uk.nhs.digital.nhsconnect.lab.results.model.edifact.ServiceProviderCommentFreeText;
+import uk.nhs.digital.nhsconnect.lab.results.model.edifact.TestStatus;
+
+/**
+ * Provides information about laboratory investigation results.
+ * <p>
+ * Segment group 18: {@code GIS-INV-SEQ-RSL-STS-DTM-FTX-RFF-SG19-SG20}
+ * <p>
+ * Parents: {@link uk.nhs.digital.nhsconnect.lab.results.model.edifact.Message Message}
+ * &gt; {@link ServiceReportDetails}
+ * &gt; {@link InvestigationSubject}
+ */
+@SuppressWarnings("OptionalUsedAsFieldOrParameterType")
+public class LabResult extends SegmentGroup {
+    public static final String INDICATOR = DiagnosticReportCode.KEY;
+
+    // GIS
+    @Getter(lazy = true)
+    private final DiagnosticReportCode diagnosticReportCode =
+        DiagnosticReportCode.fromString(extractSegment(DiagnosticReportCode.KEY));
+
+    // INV+MQ
+    @Getter(lazy = true)
+    private final LaboratoryInvestigation laboratoryInvestigation =
+        LaboratoryInvestigation.fromString(extractSegment(LaboratoryInvestigation.KEY_QUALIFIER));
+
+    // SEQ?
+    @Getter(lazy = true)
+    private final Optional<SequenceDetails> sequenceDetails =
+        extractOptionalSegment(SequenceDetails.KEY)
+            .map(SequenceDetails::fromString);
+
+    // RSL+NV?
```

Comment:
These were to indicate to me that they were optional. Key on its own is a required singleton; question mark indicates optional, otherwise range given by `{min,max}`. They were useful to me while playing the story - could be removed now if people find them useless/distracting

Reply:
I'd say remove them, but if they're not useless/distracting for others, I'm happy to keep them